.github/workflows/build-test.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Build & Test"
-on:  # yamllint disable-line rule:truthy
+on: # yamllint disable-line rule:truthy
```

Comment:
this should cause a warning from yamllint -- this is a regression

Reply:
Weird; my YAML formatter is changing it